src/styles/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+  justify-content: center;
+  &__star {
+    height: 16px;
+    width: 16px;
+    background-position: left;
+    background-image: url("../images/star.svg");
+    background-repeat: no-repeat;
+    &:not(:last-child) {
+      margin-right: 4px;
+    }
+  }
+  &--#{$starsCount} .stars__star:nth-child(-n + #{$starsCount}) {
```

Comment:
This code should work for all modifier. Use loop here
![image](https://github.com/mate-academy/layout_product-cards/assets/70009205/94b8e3b8-3c0c-4ccb-90cb-0e2332b389fe)
![image](https://github.com/mate-academy/layout_product-cards/assets/70009205/66b3fad7-517c-4d38-9252-77087d53b2b5)